### PR TITLE
Move `README.md` content to the doc site

### PR DESCRIPTION
**Pull request by jingtang10**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

**Description**
- Move the duplicated content from the README.md file to the doc site
- Add community page in the doc site to centralize all community information
- Move roadmap page to the top level navigation (it was nested under contributors)

**Alternative(s) considered**
NA

**Type**
Documentation

**Screenshots (if applicable)**

**Checklist**

- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://google.github.io/android-fhir/contrib/) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Review by vorburger — CHANGES_REQUESTED** [review]

File: docs/community.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+> Google Meet joining info
+> Video call link: https://meet.google.com/woo-hecp-ufr
+> Or dial: (GB) +44 20 3956 2102 PIN: 937 593 189#
+> More phone numbers: https://tel.meet/woo-hecp-ufr?pin=8022014824840
+
+Subscribe to
+the [OHS Developer Community Calendar](https://calendar.google.com/calendar/u/0?cid=Y19lMjAzMzkyMjY5MDFjYzM1MGE1YTVkNmEyODhkOTgxNDc0MTZlYzk0MzViNGU2OTAyMTgwNTMwM2QxNDgzNmEzQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+to see event details and other OHS events.
+
+Ask to join the [OHS Developers Google Group](ohs-developers-external@google.com) to receive the
```
[the remark this answers is not in the document]
Comment (vorburger) [reply docs/community.md]: Oh wait sorry it looks like you just did this! Resolving - please ignore.

**Review by vorburger — APPROVED** [review]
LGTM! Thanks for doing this; sorry I didn't get to it.

File: docs/community.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (jingtang10) [reply docs/community.md]: done

**Review by santosh-pingle — APPROVED** [review]

**Review by aditya-07 — APPROVED** [review]
LGTM

**Pull request merged**